src/snowcli/cli/__init__.py

```diff
@@ -57,7 +57,7 @@ def login(
         print(f"Path to snowsql config does not exist: {snowsql_config_path}")
         raise typer.Abort()
 
-    cfg_snowsql = SnowsqlConfig(snowsql_config_path)
+    cfg_snowsql = ConnectionConfigs(snowsql_config_path)
```

Comment:
IMO we should change variable name (cfg_snowsql) because we changed class name.

Reply:
Right, changed